src/parse_logcfg.c

```diff
@@ -1189,6 +1207,7 @@ static config_t logcfg_configs[] = {
     {"UNIQUE_CALL_MULTI",   NEED_PARAM, cfg_unique_call_multi},
     {"DIGI_RIG_MODE",       NEED_PARAM, cfg_digi_rig_mode},
     {"CABRILLO-(.+)",       OPTIONAL_PARAM, cfg_cabrillo_field},
+    {"RESENDCALL",          NEED_PARAM, cfg_resend_call},
```

Comment:
maybe RESEND_CALL would be more readable

Reply:
Commit 4f777a0 fixes. Please resolve this conversation if you agree the modification